tests/providers/amazon/aws/hooks/test_athena.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we really not have any existing tests for the Athena hook? (i.e. Why is this a new file?)

Reply:
That is correct.